docs/token-efficient-formatting.md

```
@@ -0,0 +1,633 @@
+            f"{status_indicator} {stack_name:<25} {status:<10} {services_display[:15]:<15}"
+        )
+    
+    return summary_lines
+```
+
+### Logs Formatting (containers/stacks)
+
+Token Efficiency Strategy: Provide a compact header with counts and a small preview; keep full logs only in structured_content.
+
+Formatted Output (container logs):
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)